src/PowerShellEditorServices.Protocol/DebugAdapter/AttachRequest.cs

```diff
@@ -21,5 +21,7 @@ public class AttachRequestArguments
         public string ProcessId { get; set; }
 
         public int RunspaceId { get; set; }
+
+        public string LocalRunspaceId { get; set; }
```

Comment:
Just asking but ... do we really need two different fields for runspace id?

Reply:
BTW if it would be more convenient if `RunspaceId` was a string, we change the type I think.  We'd just update the VSCode debug attach code to stringify the runspace id number.

Reply:
OK. I can make that change. That's pretty much the reason I added another field was that I didn't wanna mess with RunspaceId's type but I'm happy to adjust that to eliminate the new property.

Reply:
I guess I was curious how this affects existing launch.json files. Is there something we need to do to ensure that they get upgraded in some way or will a number just work?

Reply:
I just tried this. Seems to still work. It complains in the launch.json that the number should be a string but the attaching still works.